testing/influxdb/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=influxdb
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="Scalable datastore for metrics, events, and real-time analytics"
+url="https://www.influxdata.com/time-series-platform/influxdb/"
+arch="all"
+license="MIT"
+makedepends="go asciidoc xmlto"
+pkgusers="influxdb"
+pkggroups="influxdb"
+install="$pkgname.pre-install"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.zip::https://github.com/influxdata/$pkgname/archive/v$pkgver.zip
+	influxdb.initd
+	influxdb.logrotate"
+builddir="$srcdir/$pkgname-$pkgver"
+options="!strip"
```

Comment:
I did not know that, most of the packages in the repos still have !strip (at least the ones I looked at), so sorry :)